TrailInsurance/NewClaimViewController+Uploading.swift

```diff
@@ -40,14 +40,22 @@ extension NewClaimViewController {
 		} else {
 			errorDescription = "An unknown error occurred."
 		}
+		
+		if(alert.isViewLoaded){
+			alert.dismiss(animated: true)
+		}
+		alert = UIAlertController(title: "We're sorry, an error has occured. This Claim has not been saved.", message: errorDescription, preferredStyle: .alert)
+		alert.addAction(UIAlertAction(title: "Dismiss", style: .cancel, handler: { action in self.unwindToClaims()}))
```

Comment:
Just wondering if a better user experience could be to not completely go back to claims. For example if just a field was missing, the user would loose all entered data by dismissing the claim.

Reply:
René,

I thought about this. I ultimately decided to unwind back to the list, as there's nothing in the create where missing data would cause it to fail. The failure states are all on the SF side related to field visibility / setup. 

I could go either way, and it's simple enough to remove the unwind action. What're your thoughts?

Reply:
I'm fine with both. It was mostly an idea. Not all errors come from the Salesforce side of things (just figured that no network connectivity silently fails).

Reply:
I'm going to leave as-is for now.